source/triggers/database-triggers.txt

```diff
@@ -137,81 +151,130 @@ Database Triggers have the following configuration options:
+       - :guilabel:`Database`, when a change occurs on any collection in a 
+         specified database
+       - :guilabel:`Deployment`, when deployment changes occur on a specified 
+         cluster. If you select the Deployment source type, the following 
+         databases are **not watched** for changes:
 
-     - The MongoDB database that contains the watched
-       collection.
-
-   * - :guilabel:`Collection Name`
-
-     - The name of the collection that the Trigger watches for
-       change events.
-
-   * - :guilabel:`Operation Type`
-
-     - Checkboxes of :ref:`database operation types <database-event-operation-types>` 
-       that cause the Trigger to fire. Select the operation types you want the 
-       trigger to respond to.
-
-       .. warning::
-
-          Update operations executed from MongoDB Compass or the MongoDB Atlas
-          Data Explorer fully replace the previous document. As a result,
-          update operations from these clients will generate **Replace**
-          change events rather than **Update** events.
-
-   * - :guilabel:`Full Document`
-
-     - If enabled, **Update** change events include the latest
-       :manual:`majority-committed </reference/read-concern-majority/>`
-       version of the modified document *after* the change was applied in
-       the ``fullDocument`` field.
+         - The admin databases `admin`, `local`, and `config`
+         - The sync databases `__realm_sync` and `__realm_sync_<app_id>`
+         
+Depending on which source type you are using, the additional options differ. The 
+following table describes these options.
 
-       .. note::
+.. list-table::
+   :header-rows: 1
+   :widths: 15 30
 
-          Regardless of this setting:
-          
-          - **Insert** and **Replace** events always include the
-            ``fullDocument`` field.
-          
-          - **Delete** events never include the ``fullDocument`` field.
+   * - Source Type
+     - Options
 
-   * - :guilabel:`Document Preimage`
+   * - :guilabel:`Collection`
+     - |
+       - :guilabel:`Cluster Name`. The name of the MongoDB cluster that the 
+           Trigger is associated with.
+      
+       - :guilabel:`Database Name`. The MongoDB database that contains the watched
+         collection. Optional if the :guilabel:`Collection Name` is specified.
 
-     - If enabled, change events include a copy of the modified document
-       from immediately *before* the change was applied in the
-       ``fullDocumentBeforeChange`` field. All change events except for
-       **Insert** events include the document preimage. Document preimages are 
-       supported on non-sharded Atlas clusters running MongoDB 4.4+, and 
-       on sharded Atlas clusters running MongoDB 5.3 and later.
+       - :guilabel:`Collection Name`. The MongoDB collection to watch.
 
-       .. tip:: Performance Optimization
+       - :guilabel:`Operation Type`. The :ref:`database operation types 
```

Comment:
[Q] should this be `database`?

Reply:
@nathantfrank Sorry, not sure what you're asking here.

Reply:
Ahhh, I think I'm confusing myself with existing documentation details. I was thinking this should be like `collection operation types` and link to something different.